src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start work");
```

Comment:
The message 'Truck start work' could be more informative. According to the checklist, it should indicate what type of Machine is working. Consider specifying that a Truck is doing some specific truck-related work.